crates/matrix-sdk/src/docs/encryption.md

```diff
@@ -166,23 +166,23 @@ to work.
 
 1. Make sure the `e2e-encryption` feature is enabled.
 2. To persist the encryption keys, you can use [`ClientBuilder::store_config`]
-   or one of the other `_store` methods on [`ClientBuilder`].
+   or one of the `_store` methods on [`ClientBuilder`].
 
 ## Restoring a client
 
-Restoring a Client is relatively easy, still some things need to be kept in
-mind before doing so.
+Restoring a Client is relatively easy, but there some things that need to be
```

Comment:
missing word here?

Reply:
What word is missing? It reads OK to me, unless my brain is skipping something?

Reply:
Maybe it's an English idiom I don't know, but I would have expected `but there *are* some things` or something like that?

Reply:
Ha, my brain totally inserted that word!

Reply:
Fixed in f18129849f